style.css

```diff
@@ -0,0 +1,62 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  min-height: 100vh;
+  background: rgb(17, 17, 17);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.frem {
+  width: 16rem;
+  height: 16rem;
+  overflow: hidden;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
```

Comment:
```suggestion
  width: 16rem;
  height: 16rem;

  overflow: hidden;

  display: flex;
  justify-content: center;
  align-items: center;
  
  position: relative;
```

Reply:
Thankyou @Tauqeer-Ahmed-99 for showing how to group similar properties, now I got the clear Idea.